### PR TITLE
Update to new JSON from endpoints

### PR DESCRIPTION
**Pull request by puffnfresh**
The responses changed slightly since the preview. This commit updates it
to what the service currently responds with.

Biggest change is that there is no stream identifier, instead each
stream is now given an ARN.

The README.md now prints a successful response.

### CODE REVIEW

**Comment by puffnfresh** [comment]
@jonsterling a review would be useful, I'll use this for a while then upload to Hackage if it looks good.

**Comment by jonsterling** [comment]
@puffnfresh I wonder, would it be possible to use the [`Arn` type from `aws-general`](http://hackage.haskell.org/package/aws-general-0.2.2/docs/Aws-General.html#t:Arn)?

**Comment by puffnfresh** [comment]
@jonsterling that worked out well, thanks!

**Comment by jonsterling** [comment]
Great! Thanks for taking the time to update this and get the library working again. This was one of those projects that they told me to do at AlephCloud/PivotCloud but we never actually used (there were so many of those, hahaha).

File: src/Aws/DynamoDb/Streams/Types.hs [hunk 1]
```diff
@@ -1092,15 +1096,84 @@ _StatusDisabled =
       fro = either pure (const $ pure StatusDisabled)
 {-# INLINE _StatusDisabled #-}
 
+data Stream
```
Comment (jonsterling) [inline src/Aws/DynamoDb/Streams/Types.hs:1099]: Is this type used anywhere? I probably just missed it

Comment (puffnfresh) [inline src/Aws/DynamoDb/Streams/Types.hs:1099]: Yes, ListStreamsResponse now gives a Stream, not just the identifier. You can also see that Stream is just a subset of the fields in StreamDescriptor:

https://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_Stream.html

Comment (jonsterling) [inline src/Aws/DynamoDb/Streams/Types.hs:1099]: Oh, cool! :+1:

**Comment by jonsterling** [comment]
I'm happy with the code-changes here; I'll merge now, and you can play with it and release whenever you're ready.

**Pull request merged**